src/net.cpp

```diff
@@ -1776,8 +1776,9 @@ void CConnman::CreateNodeFromAcceptedSocket(std::unique_ptr<Sock>&& sock,
                                             const CAddress& addr_bind,
                                             const CAddress& addr)
 {
-    int nInbound = 0;
-    int nMaxInbound = nMaxConnections - m_max_outbound;
+    int nInbound{0};
+    int nForced{0};
```

Comment:
311902f2cf94ee0e11ed34a1373db42dbc218b20

No need to use legacy naming patterns for new variables :)

Reply:
hmmm I'm gonna keep it actually because later in this function is a boolean called `forced` so, I think this is legible